src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -4,14 +4,13 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.Comparator;
 import java.util.Iterator;
 import java.util.List;
-
```

Comment:
This line shouldn't be removed as we are importing packages from different sources (one external one internal). It doesn't follow the coding standards if you combine them into one group.